src/Common/src/Microsoft/Win32/SafeHandles/SafeHandleCache.cs

```diff
@@ -18,7 +18,7 @@ internal static class SafeHandleCache<T> where T : SafeHandle
         /// <summary>
         /// Gets a cached, invalid handle.  As the instance is cached, it should either never be Disposed
         /// or it should override <see cref="SafeHandle.Dispose(bool)"/> to prevent disposal when the
-        /// instance is the <see cref="InvalidHandle"/>.
+        /// instance is the <see cref="C:InvalidHandle"/>.
```

Comment:
❕ This is not a correct type reference. You need to fully qualify items in output form if you use a prefix. Applies to a total of three references in this file.

Reply:
This reference looks a tricky one as different child classes has different InvalidHandle properties:
[SafeCryptoHandles](https://github.com/dotnet/corefx/blob/a10890f4ffe0fadf090c922578ba0e606ebdd16c/src/System.Security.Cryptography.Csp/src/System/Security/Cryptography/SafeCryptoHandles.cs#L93)
[SafeX509ChainHandle](https://github.com/dotnet/corefx/blob/a10890f4ffe0fadf090c922578ba0e606ebdd16c/src/System.Security.Cryptography.X509Certificates/src/Microsoft/Win32/SafeHandles/SafeX509ChainHandle.cs#L19)
And there's not InvalidHandle class or struct.
We can rephrase the not to something like this:
`Gets a cached, invalid handle.  As the instance is cached, it should either never be Disposed
 or it should override <see cref="SafeHandle.Dispose(bool)"/>to prevent disposal when the instance represents an invalid handle: <see cref="C:System.Runtime.InteropServices.SafeHandle.IsInvalid"/> returns <see language="true"/>`